tests/foreman/ui/test_contentview.py

```diff
@@ -173,6 +173,7 @@ def test_positive_end_to_end(self):
             self.assertIsNotNone(self.content_views.wait_until_element
                                  (common_locators['alert.success']))
 
+    @skip_if_bug_open('bugzilla', 1297308)
     @run_only_on('sat')
     @tier2
     def test_positive_add_puppet_module(self):
```

Comment:
@sthirugn if I understood you correctly, you're suggesting that we should have an automated test specifically for the scenario that lists/displays puppet modules. Is that correct? If so, a bug +1 from me to either checking if we already have a test or to add one :)